indicators/ionos-45d7f514.yml

```diff
@@ -0,0 +1,19 @@
+title: IONOS Phishing Kit 45d7f514
+description: |
+    This phishing kit targets `IONOS` customers.
+    It uses a unique IMGUR URL to host the IONOS 
+    logo image file.
+
+references:
+  - https://urlscan.io/result/45d7f514-ddb6-48e7-8d57-22b015af83ec
+  - https://urlscan.io/result/f1a005dd-8bc6-41fe-b9b4-aed1aa4134cc
+
+detection:
+
+    pageTitle:
+        title: "Webmail Login | IONOS by 1&1"
+  
+    ionosLogo:
+        requests|contains: "i.imgur.com/jyliqfL.png"
```

Comment:
nit: I reckon this could be `requests: "https://i.imgur.com/jyliqfL.png"` but they're equivalent for real-world cases (no site is ever going to request a URL that matched this but wasn't actually imgur e.g. `https://example.com/i.imgur.com/jyliqfL.png`)

Reply:
That is a valid point, I have seen some phishing kits employ the method you have mentioned.